Package.swift

```diff
@@ -143,7 +143,11 @@ let package = Package(
         Target(
             name: "XcodeprojTests",
             dependencies: ["Xcodeproj", "TestSupport"]),
-    ])
+    ],
+    exclude: [
+        "Tests/PackageLoadingTests/Inputs",
```

Comment:
@ddunbar Do you prefer these file in fixture instead? This patch mandates that manifests end in .swift extension which are considered as sources to convention system so we either add the inputs dir in exclude or move them to fixtures.